src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/pricing/importer/PriceImporter.kt

```diff
@@ -27,9 +29,11 @@ class PriceImporter(private val priceRepo: PriceRepository,
             Supplier.SERCO -> sercoPrices.get().use { import(it, Supplier.SERCO) }
             Supplier.GEOAMEY -> geoameyPrices.get().use { import(it, Supplier.GEOAMEY) }
         }
+
+        logger.info("Supplier $supplier prices import finished in '${(System.currentTimeMillis() - start) / 1000}' seconds.")
```

Comment:
These can be removed if need be.  Added for now to make it a bit easier to see how long this part takes.